language/transaction-builder/src/lib.rs

```diff
@@ -396,9 +396,10 @@ encode_txn_script! {
 encode_txn_script! {
     name: encode_tiered_mint,
     type_arg: coin_type,
-    args: [nonce: U64, designated_dealer_address: Address, mint_amount: U64, tier_index: U64],
+    args: [nonce: U64, designated_dealer_address: Address, mint_amount: U64, tier_index: U64, approval_timestamp: U64],
     script: TieredMint,
     doc: "Mints 'mint_amount' to 'designated_dealer_address' for 'tier_index' tier.\
+          Tier based on time of AOS approval (approval_timestamp unixtime) of operation.
```

Comment:
Libra time is (somewhat oddly) Unix time in microseconds--might be worth mentioning that here and/or double-checking that this is what AOS is doing.

Reply:
ah @dmitri-perelman would aos prefer unix_time in seconds? I can make that change here

Reply:
No preferences, this code is not written yet :). If all the other Libra timestamps are using microseconds, we should probably use microseconds here as well...